### PR TITLE
[TTAHUB-3097] restore transactions for workers

### PR DESCRIPTION
**Pull request by GarrettEHill**
## Description of change

This reverts commit f749515a6495a7db8339f2f6658a433b772ce94f.

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-3097

### Every PR

- [ ] Meets issue criteria
- [ ] JIRA ticket status updated
- [ ] Code is meaningfully tested
- [ ] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [ ] API Documentation updated
- [ ] Boundary diagram updated
- [ ] Logical Data Model updated
- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions
- [ ] UI review complete

### Before merge to main

- [ ] OHS demo complete
- [ ] Ready to create production PR

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [ ] Update JIRA ticket status

### CODE REVIEW

**Review by thewatermethod — APPROVED** [review]
Looks good. Was able to add files to the queue for scanning and confirmed emails were delivered. Might be a good idea to wait for a second review before merging

**Comment by GarrettEHill** [comment]
@nvms @AdamAdHocTeam @kryswisnaskas @hardwarehuman @kcirtapfromspace 
Could I get a second review, per Matts request?

**Review by nvms — APPROVED** [review]

**Pull request merged**
